Integrations/Stealthwatch_Cloud/Stealthwatch_Cloud.py

```diff
@@ -487,34 +487,41 @@ def list_sessions_command():
 
 
 def fetch_incidents():
+    date_format = "%Y-%m-%dT%H:%M:%SZ"
 
     list_params = {
-        "order_by": 'newest',
+        "ordering": 'created',
         "limit": 100
     }
     final_alerts = []
+    last_fetch_string = demisto.getLastRun().get('last_fetch_time', None)
```

Comment:
we must consider the possibility that `ids` is set in the last run object and continue from there with the new logics.
otherwise this will be problematic during content update when an instance is configured.

Reply:
content update need to be manually tested using selfservice instance